logstash-core.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = "http://www.elastic.co/guide/en/logstash/current/index.html"
   gem.license       = "Apache License (2.0)"
 
-  gem.files         = Dir.glob(["lib/**/*.rb", "locales/*"])
+  gem.files         = Dir.glob(["lib/logstash-core.rb", "lib/logstash/**/*.rb", "spec/**/*.rb", "locales/*"])
```

Comment:
in the logstash-core gem we specifically **just** want the `lib/logstash-core.rb` and what is **under** `lib/logstash` not anything else in `lib/`.